x-pack/filebeat/module/rabbitmq/log/test/test.log

```diff
@@ -0,0 +1,78 @@
+2019-04-03 11:13:15.076 [info] <0.8.0> Log file opened with Lager
```

Comment:
It seems the exepected output files are missing. You can generate them with `GENERATE=1`